[Part-1] Adding integration tests with null and empty string

rahulKQL opened the pull request.

pmakani commented on bigtable-client-core-parent/bigtable-hbase-integration-tests-common/src/test/java/com/google/cloud/bigtable/hbase/TestPutGetDelete.java:
```diff
@@ -64,4 +69,129 @@ public void testPutGetDelete() throws IOException {
+      actualError = ex;
+      // TODO(rahulkql): Verify if this is bcoz of family not found.
+      System.out.println("Verify if this is bcoz of family not found.");
+      ex.printStackTrace();
```
is it intended ?

rahulKQL replied:
Thanks for review!
It was for test purpose, I forgot to remove those

athakor commented on bigtable-client-core-parent/bigtable-hbase-integration-tests-common/src/test/java/com/google/cloud/bigtable/hbase/AbstractTestCheckAndMutate.java:
```diff
@@ -606,4 +610,153 @@ public void testCompareOpsVersions() throws Exception {
+    byte[] rowKey = dataHelper.randomData("rowKey-");
+    byte[] qualifier = dataHelper.randomData("qualifier-");
+    byte[] value = dataHelper.randomData("test-data");
+
```
```javascript
    Table table = getDefaultTable();
    byte[] rowKey = dataHelper.randomData("rowKey-");
    byte[] qualifier = dataHelper.randomData("qualifier-");
    byte[] value = dataHelper.randomData("test-data");
```
Is it possible to declare above fields globally ? because i have seen you have used same fields in `testCheckAndDeleteWithNullInputs`,`testCheckAndMutateWithNullInputs `and `testCheckAndMutateWithNullInputs`.

rahulKQL replied:
Thanks a lot for review!!
I followed the same pattern as the existing test cases. I will check n will try to remove duplicate lines as much as possible.

rahulKQL committed:
Adding fixes for HBase Table API

rahulKQL committed:
fix: verification for Batch.Callback never being called in case of Empty list

rahulKQL committed:
test: updated batch/put methods with more scenarios

Addressing feedback comments to include more tests scenarios for null
and empty test cases.
